pkg/spammer/spammer.go

```diff
@@ -70,7 +70,7 @@ func (s *Spammer) DoSpam(shutdownSignal <-chan struct{}) (time.Duration, time.Du
 	iotaMsg := &iotago.Message{
 		NetworkID: s.networkID,
 		Parents:   tips.ToSliceOfArrays(),
-		Payload:   &iotago.Indexation{Index: indexation, Data: []byte(messageString)},
+		Payload:   &iotago.Indexation{Index: []byte(indexation), Data: []byte(messageString)},
```

Comment:
Can you limit the size of `index` and `indexSemiLazy` in the plugin where it is loaded from the config to max 64 bytes, or maybe better here in the package?